### DISCUSSION TITLE
Trying to edit 16k polygon takes 3-4 seconds

### DISCUSSION THREAD

**Discussion by Sparckmen** [geoman-io/leaflet-geoman#1507] [Questions & Help]
I'm trying to edit polygon with 16k points. After moving the point the map freezes for about 3-4 seconds. How can I improve perfomance? Sandbox: https://codesandbox.io/p/sandbox/quirky-panna-nwm6h7

**Comment by Falke-Design** [comment]
You are already using the available improvements with markerlimit option in the free version. With the pro version you would have a few more options: https://www.geoman.io/docs/modes/edit-mode

limitMarkersToZoom: Shows markers when under the given zoom level.
limitMarkersToViewport: Shows only markers in the viewport.
limitMarkersToClick: Shows markers only after the layer was clicked
